packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -170,6 +171,16 @@ export class ProjectService {
                     async () =>
                         this.projectModel.create(user.organizationUuid, data),
                 );
+
+                // Give admin user permissions to user who created this project if it is not an admin already
+                if (user.ability.cannot('manage', 'Project') && user.email) {
```

Comment:
Interesting, I assumed we would add the project admin role even for users that are org admins 🤔 
The question we need to answer is: If someone removed their Org admin role should they lose access to the projects they created ? 
cc: @owlas

Reply:
those users will automatically get access , so I'm not adding a redundant permission here, but you made a great point about that use case :eyes: